libnavigation-core/build.gradle

```diff
@@ -41,6 +41,7 @@ android {
 
 dependencies {
     implementation dependenciesList.mapboxAnnotations
+    api dependenciesList.mapboxCommon
```

Comment:
Does this need to be `api` so it's accessible from `libnavigation-ui`?

Reply:
Yes, specifically for the `ReplayLocationEngine` that takes the logger as an argument. I'm unsure whether we should use a transitive dependency for this, or make UI lib properly instantiate the module.

Reply:
> I'm unsure whether we should use a transitive dependency for this, or make UI lib properly instantiate the module.

That's right, probably in this case we can have both referenced on each module as `implementation` 🤔

Reply:
On the other hand, we're adding navigation base (which also contains interface definitions) as `api`, so by adding common (which mostly serves the same purpose) as `api` we are consistent.